CMakeLists.txt

```
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.14...3.22)
 project(qusat
         LANGUAGES CXX
         VERSION 0.1.0
-        DESCRIPTION "A tool for utilizing SAT in quantum computing"
+        DESCRIPTION "A tool for utilizing SAT in Quantum Computing"
```

shouldn't this then be 
```suggestion
        DESCRIPTION "A Tool for Utilizing SAT in Quantum Computing"
```
i am fine with keeping it the old way.